cmd/server.go

```diff
@@ -6,7 +6,7 @@ import (
 	"net"
 	"net/http"
 
-	"github.com/envoyproxy/go-control-plane/envoy/api/v2"
+	v2 "github.com/envoyproxy/go-control-plane/envoy/api/v2"
```

Comment:
nitpick: it will be helpful further in code if it is `envoyv2` or something similar instead `v2`

Reply:
confusing point is `api/v2` and then there is `discovery/v2` hence :)

Reply:
The package names are aliased to the old package names that the envoy go-control-plane used before they updated them to be something like `envoy_api_v2`, e.g: https://github.com/envoyproxy/go-control-plane/blob/51b27c23cab9d18a020bf06acb0ddd00d23a2d77/envoy/api/v2/discovery.pb.go#L4

I aliased them to save making more code changes as part of this PR, I agree that some of the shorter aliases could be clearer (which is probably why envoy have done that themselves), but I think it would make it more confusing to see what has been changed in order to update the config yggdrasil sends to envoy